From b56632066bf11e67aa63b4a31d3fe8007db21b0b Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Fri, 20 Mar 2015 22:47:37 +0800
Subject: [PATCH] MLK-10384-4 ARM: dts: add a new dts file for EPDC on i.MX7D
 SDB

Add a new DTS file for EPDC to avoid conflict with ENET1/2.

Signed-off-by: Robby Cai <r63905@freescale.com>
---
 arch/arm/boot/dts/Makefile           |  3 ++-
 arch/arm/boot/dts/imx7d-sdb-epdc.dts | 22 ++++++++++++++++++++++
 2 files changed, 24 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index f79e0ca..cd50a95 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -214,7 +214,8 @@ dtb-$(CONFIG_ARCH_MXC) += \
 	imx7d-12x12-arm2-flexcan.dtb \
 	imx7d-19x19-arm2.dtb \
 	imx7d-sdb.dtb \
-	imx7d-sdb-enet.dtb
+	imx7d-sdb-enet.dtb \
+	imx7d-sdb-epdc.dtb
 dtb-$(CONFIG_ARCH_MXS) += imx23-evk.dtb \
 	imx23-olinuxino.dtb \
 	imx23-stmp378x_devb.dtb \
diff --git a/arch/arm/boot/dts/imx7d-sdb-epdc.dts b/arch/arm/boot/dts/imx7d-sdb-epdc.dts
new file mode 100644
index 0000000..892b6a8
--- /dev/null
+++ b/arch/arm/boot/dts/imx7d-sdb-epdc.dts
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2015 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx7d-sdb.dts"
+
+&fec1 {
+	status = "disabled";
+};
+
+&fec2 {
+	status = "disabled";
+};
+
+&epdc {
+        status = "okay";
+};
+
-- 
1.8.0

